### DISCUSSION TITLE
[Bug] Cannot set application URL

### DISCUSSION THREAD

**Discussion by cberg18** [sct/overseerr#2054] [Help] [answered]
Description
Cannot set application URL

Version
1.25.0

Steps to Reproduce
Go to Settings -> General
Add an application URL
Receive error
Expected Behavior
URL added successfully.

Screenshots
![image](https://user-images.githubusercontent.com/9115073/132083054-c76d4019-cb15-407b-87f4-f7cd58d3c746.png)

Device
What device were you using when you encountered this issue? Please provide this information to help us reproduce and investigate the bug.

Desktop
Win 10
Chrome, Firefox

**Comment by danshilm** [comment, marked as answer]
Like the error message says, `overseerr.domain.tld` is not a valid URL, you need to also include the protocol, be it `http` or `https`.
